README.md

```diff
@@ -16,26 +14,6 @@ If you publish your prototypes online, they **must** be protected by a [username
 
 You must protect user privacy at all times, even when using prototypes. Prototypes made with the kit look like GOV.UK, but do not have the same security provisions. Always make sure you are handling user data appropriately.
 
-## Installation instructions
-
-- [Installation guide for new users (non technical)](https://govuk-prototype-kit.herokuapp.com/docs/install/introduction)
-- [Installation guide for developers (technical)](https://govuk-prototype-kit.herokuapp.com/docs/install/developer-install-instructions)
-
-## Guides
-
-1. [Setting up git](https://govuk-prototype-kit.herokuapp.com/docs/setting-up-git)
-2. [Publishing on the web (Heroku)](https://govuk-prototype-kit.herokuapp.com/docs/publishing-on-heroku)
-3. [Using GOV.UK Verify](https://govuk-prototype-kit.herokuapp.com/docs/using-verify)
-
-## Other documentation
-
-- [Prototype kit principles](https://govuk-prototype-kit.herokuapp.com/docs/principles)
-- [Making pages](https://govuk-prototype-kit.herokuapp.com/docs/making-pages)
-- [Writing CSS](https://govuk-prototype-kit.herokuapp.com/docs/writing-css)
-- [Updating the kit to the latest version](https://govuk-prototype-kit.herokuapp.com/docs/updating-the-kit)
-- [Tips and tricks](https://govuk-prototype-kit.herokuapp.com/docs/tips-and-tricks)
-- [Creating routes (server-side programming)](https://govuk-prototype-kit.herokuapp.com/docs/creating-routes)
-
 ## Community
 
 We have two Slack channels for the Prototype kit. You'll need a government email address to join them.
```

Comment:
It's not very helpful - it doesn't tell people where to find the channels. We can point them at the url to join slack.